src/sidebar/store/modules/toast-messages.js

```diff
@@ -1,5 +1,7 @@
 import * as util from '../util';
 
+/** @typedef {import("../../services/toast-messenger").ToastMessage} ToastMessage */
```

Comment:
As noted in `services/toast-messenger`, I'd prefer the typedef to be moved to this file to preserve the overall dependency structure.

Reply:
roger that